test/extensions/filters/http/ext_proc/BUILD

```diff
@@ -39,6 +39,7 @@ envoy_extension_cc_test(
     }),
     extension_names = ["envoy.filters.http.ext_proc"],
     rbe_pool = "2core",
+    shard_count = 8,
```

Comment:
I really want to know why this test is so slow you need to use mutliple cores and 8 shards. I think I asked this before but didn't see the answer.

Reply:
I quickly check the trace log timestamp. it looks the tests themselves are very fast. i.e, <5ms in a local setup. It's the test initialization consumed most of the time (>100ms in the same local setup).  And this is the case for existing tests as well, like the very basic SimpestPost test: https://github.com/envoyproxy/envoy/blob/c414d2870d2dad85bb9d14e6c60d5507a44c3490/test/extensions/filters/http/ext_proc/filter_test.cc#L624